server-common/src/main/java/org/apache/gravitino/server/authorization/expression/AuthorizationExpressionConverter.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.server.authorization.expression;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.gravitino.server.authorization.MetadataFilterHelper;
+
+/**
+ * Convert the authorization expression into an executable expression, such as OGNL expression, etc.
+ */
+public class AuthorizationExpressionConverter {
+
+  /** Match authorization expressions */
+  private static final Pattern PATTERN = Pattern.compile("([A-Z_]+)::([A-Z_]+)");
```

Comment:
Could you give an example?

Reply:
hi @hdygxsj please add a unit test to cover this pattern, thanks.